tutorials/containers/deploying-storageos-on-kubernetes/deploying-storageos-on-kubernetes.md

```diff
@@ -0,0 +1,333 @@
+  name: wordpress
+  labels:
+    app: wordpress
+spec:
+  selector:
+    matchLabels:
+      app: wordpress
+      tier: frontend
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: wordpress
+        tier: frontend
+    spec:
+      containers:
+      - image: wordpress:4.8-apache
+        name: wordpress
+        env:
+        - name: WORDPRESS_DB_HOST
+          value: wordpress-mysql
+        - name: WORDPRESS_DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: mysql-pass
+              key: password
+        ports:
+        - containerPort: 80
+          name: wordpress
+        volumeMounts:
+        - name: wordpress-persistent-storage
+          mountPath: /var/www/html
+      volumes:
+      - name: wordpress-persistent-storage
+        persistentVolumeClaim:
+          claimName: wp-pv-claim
+END
+```
+
+
+## Access your new site
+Duration: 5
+
+Check the pod status:
+```
+kubectl get pods -w
+```
+You should see something like this:
+```
+NAME                               READY   STATUS    RESTARTS   AGE
+csi-rbdplugin-attacher-0           1/1     Running   1          60m
+csi-rbdplugin-bxgsk                2/2     Running   1          59m
+csi-rbdplugin-kcmdd                2/2     Running   2          35m
+csi-rbdplugin-provisioner-0        3/3     Running   0          60m
+csi-rbdplugin-zlglm                2/2     Running   1          59m
+wordpress-74db446497-2724m         1/1     Running   1          10m
+wordpress-mysql-7c795bf6dc-9t7dj   1/1     Running   0          10m
+
+```
+
+To access your new site we need to get the IP of the svc, you can do that by using `kubetl get svc`.
+
+You will then be presented with output such as this:
+
+```
+NAME         TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE
+wordpress    NodePort    10.152.183.72   <none>        80:31625/TCP   2m26s
+kubernetes   ClusterIP   10.152.183.1    <none>        443/TCP        31m
+```
+
+Now you can try to access the wordpress site by navigating to:
+
+`https://10.152.183.72/`
+
+If you encounter a blank screen, trying running `kubectl describe pod <<wordpress_pod_name>>` and accessing the site on the node's address found at the top of the describe with port 31625.
+
+You can go through the setup steps and should get something like this:
+
+![Wordpress site](images/wp-site.png)
+
+## That's all folks!
+
+Congratulations! In this tutorial you deployed StorageOS on Charmed Kubernetes and created a workload a wordpress site which used StorageOS as it's storage backend.
+
+This tutorial can be adapted for other tasks which require persistent volumes and shows you the ease of use a solution such as StorageOS offers.
+
+### Where to go from here?
+- https://microk8s.io/
+- https://jaas.ai/kubernetes
+- https://github.com/ubuntu/microk8s/issues
+- https://ubuntu.com/kubernetes/contact-us
+- https://kubernetes.io/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/
```

Comment:
Please update (Charmed Kubernetes vs MicroK8s) and add some words (e.g. "Visit", etc.) rather than just throwing the links.

Reply:
Opps